assets/javascripts/kitten/components/icons/checked-circle-icon.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react'
 
-export const CheckedCircleIcon = ({ circleColor, checkedColor, ...props }) => {
+export const CheckedCircleIcon = ({
+  circleColor,
+  checkedColor,
+  title,
+  ...props
+}) => {
   return (
     <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" {...props}>
-      <title>Checked circle</title>
+      {title && <title>Checked circle</title>}
```

Comment:
```suggestion
      {title && <title>{ title }</title>}
```

et en `defaultProp` : `Checked circle`.